From 08f6eb7384e886bcbf88f07a31611f678ad2a968 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Sat, 30 May 2026 09:15:07 +0100
Subject: [PATCH 3/3] product: Exclude AOSP sample APN list from GSI build

The apns-full-conf.xml includes thousands of carrier APNs that bloat
the GSI image. Carrier APNs are loaded from the SIM by modern modems
and Android 12+ auto-configures APNs from the carrier. The full conf
file is not needed on a GSI.
---
 target/product/gsi/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/gsi/Android.bp b/target/product/gsi/Android.bp
index 82d5705..250be15 100644
--- a/target/product/gsi/Android.bp
+++ b/target/product/gsi/Android.bp
@@ -132,7 +132,7 @@ android_filesystem_defaults {
         "TrichromeChrome",
         "Dialer",
         "LatinIME",
-        "apns-full-conf.xml",
+        
         "frameworks-base-overlays",
     ],
     multilib: {
-- 
2.54.0

